ci-operator/step-registry/openshift/e2e/openstack/perfscale/conf-securitygroup/provision/openshift-e2e-openstack-perfscale-conf-securitygroup-provision-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
We have a `provision` directory where to put OpenStack-specific provisioning steps. The idea behind the step-registry is to reuse steps: if you namespace your steps under `perfscale`, it kind of defeats the purpose.

Instead of the current step path:
```plaintext
ci-operator/step-registry/openshift/e2e/openstack/perfscale/conf-securitygroup/provision
```

I propose to put it under our regular `provision` step library, with a pattern that we could reuse in case we need to add other security groups. I suppose that the reason you need port 12865 is [netperf,](https://github.com/HewlettPackard/netperf/blob/3bc455b23f901dae377ca0a558e1e32aa56b31c4/doc/netperf.info#L413) hence I propose:

```plaintext
ci-operator/step-registry/openstack/provision/securitygroups/netperf
```